plots2_inprogress

```diff
@@ -0,0 +1,53 @@
+diff --git a/test/integration/incoming_mail_parsing_test/gmail_parsing_test.rb b/test/integration/incoming_mail_parsing_test/gmail_parsing_test.rb
```

Comment:
Maybe this is just a stray file for note-taking that we could remove, @enviro3 ?